bin/compile

```diff
@@ -344,7 +373,9 @@ cache_build() {
   cache_directories="$(get_cache_directories "$BUILD_DIR")"
 
   clear_cache "$CACHE_DIR"
-  if ! ${NODE_MODULES_CACHE:-true}; then
+  if $PNPM; then
+    echo "- pnpm (nothing to cache)"
```

Comment:
I suspect this is a dumb question, but why aren't we caching here? Looks like we do it below on L415, but then Yarn doesn't 'opt out' of caching here